src/main/java/com/bettercloud/vault/api/Logical.java

```diff
@@ -162,4 +150,29 @@ public LogicalResponse write(final String path, final Map<String, String> nameVa
         }
     }
 
+    private LogicalResponse buildResponseFromContent(RestResponse restResponse, int retryCount) throws VaultException {
```

Comment:
Why is `retryCount` being passed as a parameter?  Unless I'm caffeine-deprived and missing something... it looks like it's just returned as-is at the end, without having been actually used by this method.